packages/lib/src/vis/shared/VisCanvas.tsx

```diff
@@ -60,50 +52,42 @@ function VisCanvas(props: PropsWithChildren<Props>) {
         paddingRight: axisOffsets.right,
       }}
     >
-      {(!shouldMeasure || canvasSize) && (
-        <div
-          className={styles.canvasWrapper}
-          style={shouldMeasure ? canvasSize : undefined}
+      <div className={styles.canvasWrapper}>
```

Comment:
Perhaps this `div` is no longer needed ?

Reply:
I was able to make a bunch of improvements. I'll comment.